mesonbuild/compilers/mixins/clike.py

```diff
@@ -317,12 +317,12 @@ def _sanity_check_impl(self, work_dir: str, environment: 'Environment',
             cmdlist = [binary_name]
         mlog.debug('Running test binary command: ', mesonlib.join_args(cmdlist))
         try:
-            # fortran code writes to stdout
-            pe = subprocess.run(cmdlist, stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL)
+            mlog.debug('Sanity check run command line: ', ' '.join(cmdlist))
```

Comment:
Delete this line entirely. It's already immediately above, except better as the version immediately above uses mesonlib.join_args.

Reply:
Nit: you dropped this (and the below) in the second commit but it is still modified then reverted in the first commit.